test/unit/gateways/alelo_test.rb

```diff
@@ -70,12 +71,12 @@ def test_successful_purchase_with_provided_credentials
       decrypted = JOSE::JWE.block_decrypt(secret_key, JSON.parse(data)['token']).first
       request = JSON.parse(decrypted, symbolize_names: true)
 
-      assert_equal @options[:order_id], request[:request_id]
-      assert_equal '1.00', request[:amount]
+      assert_equal @options[:order_id], request[:requestId]
+      assert_equal 1.0, request[:amount]
```

Comment:
Is this really correct? are we sure only `.to_f` is sufficient for correct formatting? I wouldn't expect `1.0` to be proper.

Reply:
well, the money method applies the proper format but it returns '1.00' and Alelo needs float number, documentation says 'string' but based on the homologation work and the API example, I'm sending the return of amount method as a float number.

![capture___sandbox-desenvolvedor_alelo_com_br_br](https://user-images.githubusercontent.com/133580/192057317-c1cf5a5b-6b6e-4cca-8949-6dda6dfb6a5d.png)

Reply:
mmm that should not be correct, I tested on a random local test with an amount of 100 that end up being translated as 1.0
![tmux](https://user-images.githubusercontent.com/133580/192368130-79cfd3fb-fbae-4ceb-a471-2f3d5f4a5fb0.png)

Reply:
Here is what I'm seeing:
<img width="1042" alt="Screen Shot 2022-09-26 at 4 19 57 PM" src="https://user-images.githubusercontent.com/5394429/192372484-10ef76d9-a92f-40a3-8f74-5d5b048f0f12.png">

Reply:
I do see `1.0` in the request, but that is what I get for the response.

Reply:
Ahhh the beauty of Alelo's Sandbox @curiousepic , no matter what you send in the body, using the uuid '5314 ...' you are always going to get that response:


![Screen Shot 2022-09-26 at 3 28 15 PM](https://user-images.githubusercontent.com/133580/192373634-f130247f-fb3a-4888-bbfc-941ec0838d73.png)

Reply:
Aah, it looks like the sandbox always returns that amount 😩